test/controllers/api/v2/base_controller_subclass_test.rb

```diff
@@ -109,6 +109,16 @@ def setup
         assert_equal user.id, session[:user]
       end
     end
+
+    context 'disabled user cannot be authenticated' do
+      let(:user) { as_admin { FactoryBot.create(:user, :admin, disabled: true) } }
```

Comment:
It would be better to be explicit about `password` as we care about that being set on our value (dafault may change in the future)
```suggestion
      let(:user) { as_admin { FactoryBot.create(:user, :admin, disabled: true, password: 'password') } }
```